lib/SIL/InstructionUtils.cpp

```diff
@@ -277,6 +277,18 @@ SILValue swift::stripBorrow(SILValue V) {
   return V;
 }
 
+SILValue swift::getSingleNonIncidentalUse(SILValue V) {
+  SILValue singleUse;
+  for (auto *use : V->getUses()) {
+    if (isIncidentalUse(use->getUser()))
+      continue;
+    if (singleUse)
+      return nullptr;
+    singleUse = SILValue::getFromOpaqueValue(use->getUser());
```

Comment:
I don't think you want to cast a `void *` to a SILValue.

Reply:
Fixed.